<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>7 labs in 48 hours</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="create_field">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>30</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Начать игру</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_unit">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>90</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить юнита</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>140</y>
      <width>176</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="wizard">
         <property name="text">
          <string>Wizard</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="panzer">
         <property name="text">
          <string>Panzer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="swordsman">
         <property name="text">
          <string>SwordsMan</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="archer">
         <property name="text">
          <string>Archer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="centaur">
         <property name="text">
          <string>Centaur</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="spearman">
         <property name="text">
          <string>SpearMan</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>731</width>
      <height>731</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gameField"/>
   </widget>
   <widget class="QTextBrowser" name="charsObject">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>370</y>
      <width>171</width>
      <height>371</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>974</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="actionLoad"/>
   <addaction name="undoAction"/>
   <addaction name="actionSettings"/>
  </widget>
  <action name="actionSettings">
   <property name="text">
    <string>Настройки</string>
   </property>
   <property name="toolTip">
    <string>Settings</string>
   </property>
  </action>
  <action name="undoAction">
   <property name="text">
    <string>Отмена</string>
   </property>
   <property name="toolTip">
    <string>Отмена последнего действия</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранение игры</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Загрузка игры</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
